src/highlight/highlightUtil.ts

```diff
@@ -1,5 +1,12 @@
-import { Annotation, AnnotationHighlight, Rect, Type } from '../@types';
+import { Annotation, AnnotationHighlight, DOMRectMini, Rect, Type } from '../@types';
 import { checkValue } from '../utils/util';
+import { getPageNumber } from '../document/docUtil';
```

Comment:
This creates a dependency from a generic utility on a specific viewer/annotator. Should all of this code live in docUtil?

Reply:
I moved them from docUtil because of a previous comment. I think pdfjs related page number should live in docUtil and selection related functions should live in highlightUtil.

Reply:
Where the code lives is less important than the dependencies between the code. Other annotators would not be able to use this utility, since it's dependent on a specific annotator. They need to be decoupled.